cmd/web.go

```diff
@@ -41,7 +41,7 @@ and it takes care of all the other things for you`,
 		},
 		cli.StringFlag{
 			Name:  "pid, P",
-			Value: "/var/run/gitea.pid",
+			Value: "/run/gitea.pid",
```

Comment:
If we moved this value out in to a var then this could be made compile/link time settable

Reply:
```suggestion
```